spec.html

```diff
@@ -6978,12 +6978,66 @@ <h1>
     <h1>Operations on Iterator Objects</h1>
     <p>See Common Iteration Interfaces (<emu-xref href="#sec-iteration"></emu-xref>).</p>
 
+    <emu-clause id="sec-iterator-records">
+      <h1>Iterator Records</h1>
+      <p>An <dfn variants="Iterator Records">Iterator Record</dfn> is a Record value used to encapsulate an Iterator or AsyncIterator along with the `next` method.</p>
+      <p>Iterator Records have the fields listed in <emu-xref href="#table-iterator-record-fields"></emu-xref>.</p>
+      <emu-table id="table-iterator-record-fields" caption="Iterator Record Fields">
+        <table>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[Iterator]]
+            </td>
+            <td>
+              An object
+            </td>
+            <td>
+              An object that conforms to the <emu-xref href="#sec-iterator-interface">Iterator</emu-xref> or <emu-xref href="#sec-asynciterator-interface">AsyncIterator</emu-xref> interface.
```

Comment:
```suggestion
              An object that conforms to the <em>Iterator</em> or <em>AsyncIterator</em> interface.
```

Right now we refer to these interface names with italics, and don't link them. https://github.com/tc39/ecma262/issues/2486 will fix that, but in the mean time we should be consistent with existing usage.